module/mutect2-processes.nf

```diff
@@ -249,7 +220,7 @@ process run_FilterMutectCalls_GATK {
     path "*_filteringStats.tsv"
 
     script:
-    contamination = params.use_contamination_estimation ? contamination_estimation.collect { "--contamination-table '$it'" }.join(' ') : ""
+    contamination = 'NO_PATH' == contamination_estimation.collect().join(' ') ? "" : contamination_estimation.collect{ "--contamination-table '$it'" }.join(' ')
```

Comment:
Didn't understand this, I assume this will work? @yashpatel6

Reply:
It should work yes, it's basically a concise if-else statement and we switched from having a parameter to having a default of `NO_PATH` when there is no contamination table given